fixtures/analysis.py

```diff
@@ -0,0 +1,9 @@
+import json
+import pytest
+
+
+@pytest.fixture(scope="session")
+def json_analysis():
+    with open('../data/analysis.json', 'r') as file:
```

Comment:
@abrugaro I think `data/analysis.json` as the file's path should work, could you please try it?

Reply:
It is not working for me `
E       FileNotFoundError: [Errno 2] No such file or directory: 'data/analysis.json'`

Reply:
I see. Thanks for checking